addons/binding/org.openhab.binding.innogysmarthome/README.md

```diff
@@ -0,0 +1,177 @@
+| button3_count | Number | number of button pushes for button 3, increased with each push | BRC8 |
+| button4_count | Number | number of button pushes for button 4, increased with each push | BRC8 |
+| button5_count | Number | number of button pushes for button 5, increased with each push | BRC8 |
+| button6_count | Number | number of button pushes for button 6, increased with each push | BRC8 |
+| button7_count | Number | number of button pushes for button 7, increased with each push | BRC8 |
+| button8_count | Number | number of button pushes for button 8, increased with each push | BRC8 |
+| luminance | Number | Indicates the measured luminance in percent | WMD, WMD0 |
+| mold_warning | Switch | active, if the measured humidity is too low (ON/OFF) | RST | 
+| motion_count | Number | Number of detected motions, increases with each detected motion | WMD, WMDO |
+| operation_mode | String | the mode of a thermostat (auto/manual) | RST | 
+| rollershutter | Rollershutter | Controlls a roller shutter | ISR2 |
+| set_temperature | Number | Sets the target temperature in °C | RST, WRT |
+| smoke | Switch | Indicates, if smoke was detected (ON/OFF) | WSD, WSD2 |
+| switch | Switch | A switch to turn the device or variable on/off (ON/OFF) | ISS2, PSS, PSSO, VariableActuator |
+| temperature | Number | Holds the actual temperature in °C | RST, WRT |
+| window_reduction_active | Switch | indicates if a linked window is open and temperature reduced (ON/OFF)  | RST |
+
+## Thing configuration
+
+### Configuring the SmartHome Controller (SHC)
+
+The SmartHome Controller (SHC) can be configured in the PaperUI as follows:
+
+After the "innogy SmartHome Controller" is added via the Inbox, edit the controller and add the "Authorization code" by following the hints in the description. Save your changes. The SHC should now login and go online. Be sure it is connected to the internet.
+
+### Obtaining the authorization code and tokens
+
+Authorization is done as oauth2 workflow with the innogy API.
+
+To receive the auth-code, go to one of the following URLs depending on your brand and login with your credentials (you can find this link also in the SHC thing in Paper UI, if you edit it):
+* [innogy SmartHome authorization page](https://api.services-smarthome.de/AUTH/authorize?response_type=code&client_id=24635748&redirect_uri=https%3A%2F%2Fwww.ollie.in%2Finnogy-smarthome-token%2F&scope&lang=de-DE)
+* [SmartHome Austria authorization page](https://api.services-smarthome.de/AUTH/authorize?response_type=code&client_id=24635749&redirect_uri=https%3A%2F%2Fwww.ollie.in%2Fsmarthome-austria-token%2F&scope&lang=de-DE)
+* [Start SmartHome authorization page](https://api.services-smarthome.de/AUTH/authorize?response_type=code&client_id=24635750&redirect_uri=https%3A%2F%2Fwww.ollie.in%2Fstart-smarthome-token%2F&scope&lang=de-DE)
+
+You will be redirected to the webpage of the maintainer of the binding, that displays the auth-code.
```

Comment:
you will now be redirected to openhab.org

Reply:
Changed.